sycl/include/CL/sycl/triple.hpp

```diff
@@ -0,0 +1,64 @@
+//==-------------- triple.hpp - SYCL device triple descripter --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/backend_types.hpp>
+#include <CL/sycl/detail/defines.hpp>
+#include <CL/sycl/info/info_desc.hpp>
+
+#include <fstream>
+#include <iostream>
+#include <istream>
+#include <string>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+
+#define DEVICE_NUM_UNSPECIFIED -1
```

Comment:
Please, do not define internal things in a way they get into the user space. Also please use constexpr variable instead of macro.